tensorflow/compiler/xla/service/hlo_instructions.h

```diff
@@ -1217,8 +1217,12 @@ class HloCustomCallInstruction : public HloInstruction {
   void set_batch_group_count(int64 batch_group_count) {
     batch_group_count_ = batch_group_count;
   }
+  void set_has_side_effect(const bool has_side_effect) {
+    has_side_effect_ = has_side_effect;
+  }
   int64 feature_group_count() const { return feature_group_count_; }
   int64 batch_group_count() const { return batch_group_count_; }
+  const bool has_side_effect() const { return has_side_effect_; }
```

Comment:
The `const` in `const bool` is effectively meaningless in the C++ language and should be removed.